samza-core/src/main/java/org/apache/samza/runtime/AbstractApplicationRunner.java

```diff
@@ -108,8 +114,8 @@ final void writePlanJsonFile(String planJson) {
   }
 
   @VisibleForTesting
-  StreamManager buildAndStartStreamManager() {
-    StreamManager streamManager = new StreamManager(this.config);
+  StreamManager buildAndStartStreamManager(Config config) {
+    StreamManager streamManager = new StreamManager(config);
```

Comment:
nit-pick: maybe out of scope for this PR, but consider inlining the 2 line helper.

Reply:
This is for testing, since we don't (can't) inject the StreamManager into the runner/planner.